package.json

```diff
@@ -109,8 +114,7 @@
     "release": "standard-version -i HISTORY.md",
     "test": "npm run check:lint && npm run check:test"
   },
-  "homepage": "https://github.com/mongodb/node-mongodb-native",
-  "optionalDependencies": {
-    "saslprep": "^1.0.0"
+  "tsd": {
+    "directory": "test/unit/types"
```

Comment:
This is the only place that [tsd](https://github.com/SamVerschueren/tsd#readme) accepts this configuration, otherwise I would've just put it in the added test.